packages/console/src/components/ColorPicker/index.module.scss

```diff
@@ -0,0 +1,40 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  border-radius: 6px;
+  border: 1px solid var(--color-component-border);
+  outline: 2px solid transparent;
+  transition-property: outline, border;
+  transition-timing-function: ease-in-out;
+  transition-duration: 0.2s;
+  padding: _.unit(2) _.unit(3);
+  font: var(--font-body-medium);
+  position: relative;
+
+  &:focus-within {
+    border-color: var(--color-primary);
+    outline-color: var(--color-primary-80);
```

Comment:
outline-color should be `--color-focused-variant`

Reply:
updated